apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokensBar.vue

```diff
@@ -81,12 +111,26 @@ const resetSearch = () => {
 const hasSearchTerm = computed(() => searchInput.value && searchInput.value.trim().length > 0);
 
 const setFormat = (newFormat) => {
+  router.replace({ path: '/tokens/', query: { ...route.query, format: newFormat } });
   emit('changeFormat', newFormat);
 };
 
 const setTheme = (newTheme) => {
+  router.replace({ path: '/tokens/', query: { ...route.query, theme: newTheme } });
```

Comment:
We should also keep the hash in the route to allow to share a specific subcategory, for example: `https://dialtone.dialpad.com/deploy-previews/pr-337/tokens/?theme=light#color-surface`

Reply:
That's a bit more difficult as the search box and the options are on top of the page and the hash changes while going to top so It'd be removed if you go up to click `share filter`.

https://github.com/dialpad/dialtone/assets/87546543/63aa0921-9b53-4fb5-952c-c16f8e994f65

Reply:
Yeah I think in order to do this you'd have to have a share filter or similar on each category. Maybe a future upgrade since this is not trivial.

Reply:
> Yeah I think in order to do this you'd have to have a share filter or similar on each category. Maybe a future upgrade since this is not trivial.

That or making the filters fixed, that might be easier but it's more a design decision than engineering really, so let's see what our Design folks decide.

Reply:
@juliodialpad I meant when manually copying the URL. The current behavior is that this url `https://dialtone.dialpad.com/tokens/#color-border` scrolls to the color/border section. Right now the hash is lost when copying manually.

Reply:
This is the current behavior:
![tokens-curr](https://github.com/dialpad/dialtone/assets/24460973/53653ae6-a261-49bc-a9b3-c7987210272e)

In this PR:
![tokens-pr](https://github.com/dialpad/dialtone/assets/24460973/c5f436e0-c803-4298-94a6-2f285047b4b8)

Reply:
You were right! I was overriding the hash while updating the query params, is fixed now.